phpstan.neon

```diff
@@ -0,0 +1,60 @@
+# https://phpstan.org/config-reference
+# https://github.com/phpstan/phpstan-src/blob/51ba944f204bcddee7f433a92c45b80cc455cdfa/conf/config.neon#L190
+
+# Note 'always falsy for empty checks on arrays'
+# https://github.com/phpstan/phpstan/issues/6762
+
+# Multiple functions with the same name
+# https://github.com/phpstan/phpstan/issues/4114
+# https://github.com/phpstan/phpstan/issues/9314
+#
+# fix for date('m') etc issues.
+# https://github.com/phpstan/phpstan-src/pull/2888
+#
+# empty array foreach
+# https://github.com/phpstan/phpstan/issues/10345
+# https://github.com/phpstan/phpstan/issues/2560
+#
+# https://github.com/phpstan/phpstan/issues/7773
+# Call to function is_string() with string|false will always eval false
+#
+# Easy fixes
+# https://github.com/phpstan/phpstan/milestone/8
+#
+parameters:
+    fileExtensions:
+        - php
+        - inc
+    paths:
+        - .
+    excludePaths:
```

Comment:
Probably `pinc/3rdparty` too.

Reply:
I'm 50/50 on this one. On the one hand, adding it to the exclude path means we don't have to fix 3rd party code, which is good. On the other hand, it means we don't _see_ it either, and get a handful of errors in our interfacing code like
`Class BBDiffFormatter extends unknown class DiffFormatter`